testData/solutions/win32.json5

```diff
@@ -5435,108 +5402,76 @@
-                                            "match": {
-                                                "White": "",
-                                                "Black": "%SystemRoot%\\cursors\\cross_r.cur",
-                                                "ReverseBlack": "%SystemRoot%\\cursors\\cross_i.cur"
-                                            },
-                                            "noMatch": {
-                                                "outputValue": ""
-                                            }
-                                        }
-                                    }
-                                }, {
-                                    "upperBound": 0.666,
-                                    "output": {
-                                        "transform": {
-                                            "type": "fluid.transforms.valueMapper",
-                                            "defaultInputPath": "http://registry\\.gpii\\.net/common/cursorColor",
-                                            "match": {
-                                                "White": "%SystemRoot%\\cursors\\cross_m.cur",
-                                                "Black": "%SystemRoot%\\cursors\\cross_rm.cur",
-                                                "ReverseBlack": "%SystemRoot%\\cursors\\cross_im.cur"
-                                            },
-                                            "noMatch": {
-                                                "outputValue": "%SystemRoot%\\cursors\\cross_m.cur"
-                                            }
-                                        }
-                                    }
-                                }, {
-                                    "output": {
-                                        "transform": {
-                                            "type": "fluid.transforms.valueMapper",
-                                            "defaultInputPath": "http://registry\\.gpii\\.net/common/cursorColor",
-                                            "match": {
-                                                "White": "%SystemRoot%\\cursors\\cross_l.cur",
-                                                "Black": "%SystemRoot%\\cursors\\cross_rl.cur",
-                                                "ReverseBlack": "%SystemRoot%\\cursors\\cross_il.cur"
-                                            },
-                                            "noMatch": {
-                                                "outputValue": "%SystemRoot%\\cursors\\cross_l.cur"
-                                            }
-                                        }
-                                    }
-                                }
-                            ]
-                        }
-                    },
+                    // TODO: Remove this or add as a supported setting.
```

Comment:
@amb26, there were two settings like this that were inferred from common terms but not defined.  I can either prune them or add them to supportedSettings, whichever you prefer.

Reply:
Agreed to reconstruct these.